### PR TITLE
Add conditional stage to upload the generated nuget package to the UploadPackageToPreReleaseFeed

### LINKED ISSUE #396: Onboard Nodejs worker to be built as part of Integration Tests

### PR DESCRIPTION
**Pull request by Francisco-Gamino**
Resolves https://github.com/Azure/azure-functions-nodejs-worker/issues/396

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -119,7 +119,7 @@ jobs:
       failTaskOnFailedTests: true
 
 - job: BuildArtifacts  
-  condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/master'), eq(variables['Build.SourceBranch'], 'refs/heads/v2.x')))
+  condition: and(succeeded(), or(eq(variables['Build.SourceBranch'], 'refs/heads/master'), eq(variables['Build.SourceBranch'], 'refs/heads/v2.x'), eq(variables['UPLOADPACKAGETOPRERELEASEFEED'], true)))
```
Comment (alrod) [inline azure-pipelines.yml:122]: Do we need to add spaces for the variable name `UPLOAD_PACKAGE_TO_PRERELEASE_FEED`?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -137,3 +137,12 @@ jobs:
       PathtoPublish: '$(Build.ArtifactStagingDirectory)'
       ArtifactName: 'drop'
       publishLocation: 'Container'
+  - task: NuGetCommand@2
+    condition: eq(variables['UPLOADPACKAGETOPRERELEASEFEED'], 'True')
+    inputs:
+      command: 'push'
+      packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;!$(Build.ArtifactStagingDirectory)/**/*.symbols.nupkg'
+      nuGetFeedType: 'internal'
+      publishVstsFeed: 'e6a70c92-4128-439f-8012-382fe78d6396/f37f760c-aebd-443e-9714-ce725cd427df'
```
Comment (alrod) [inline azure-pipelines.yml:146]: what is the link to the feed (just curious)? do we expect the package will be overwriten with latest Microsoft.Azure.Functions.NodeJsWorker x.x.x on each CI run?

Comment (Francisco-Gamino) [reply azure-pipelines.yml:146]: The package will be upload it to `https://azfunc.visualstudio.com/Azure%20Functions/_packaging?_a=feed&feed=AzureFunctionsPreRelease`. If no changes have been done to the language worker code base, then the package version will not be updated. Also, the tasks is configure to skip duplicates as shown below.

image

File: azure-pipelines.yml (hunk 1 above)
Comment (Francisco-Gamino) [reply azure-pipelines.yml:122]: According to the help doc, we do not have to add spaces to the variable. Please see below:

image

**Review by alrod — APPROVED** [review]

**Pull request merged**